apps/openmw/mwscript/miscextensions.cpp

```diff
@@ -1254,6 +1254,7 @@ namespace MWScript
                     msg << "[Deleted]" << std::endl;
 
                 msg << "RefID: " << ptr.getCellRef().getRefId() << std::endl;
+                msg << "Ptr: " << ptr.getBase() << std::endl;
```

Comment:
The string like `Ptr: 0x55df90966fb0` is not informative enough for modmakers, IMO.
How about "Memory address" or something like that?

Reply:
Done